src/dashboard/components/navbar_component.py

```diff
@@ -99,3 +99,14 @@ def navbar_component() -> Component:
 def update_navbar(path_name: str) -> list[Component]:
     """Update the selected navbar item based on the current url."""
     return generate_navbar_items(dash.page_registry, path_name)
+
+
+@callback(
+    Output("main-navbar", "className"), Input("url", "pathname"), State("main-navbar", "className")
+)
+def show_navbar(path_name: str, class_name: str) -> list[Component]:
```

Comment:
I'm not sure this callback should be in navbar_component.py. Maybe the login page controller would be a more suitable place?

In any case, the classname functionality should be in another functionality. There is a `toggle_class` implementation in `dev` that might be useful. It might be best to extract the different branches of that function into separate functions. `set_class`, `unset_class`?

Reply:
Going to put it as an issue :+1: